pkgs/development/python-modules/py-mini-racer/default.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
+  pname = "py-mini-racer";
+  version = "0.6.0";
```

```suggestion
  version = "0.6.0";
  pyproject = true;
```

and configure `build-system`.